src/core/server/core_usage_data/core_usage_data_service.ts

```diff
@@ -0,0 +1,211 @@
+              es.ssl.certificateAuthorities
+            ),
+            certificateConfigured: isConfigured.string(es.ssl.certificate),
+            keyConfigured: isConfigured.string(es.ssl.key),
+            verificationMode: es.ssl.verificationMode,
+            truststoreConfigured: isConfigured.record(es.ssl.truststore),
+            keystoreConfigured: isConfigured.record(es.ssl.keystore),
+          },
+        },
+        http: {
+          basePathConfigured: isConfigured.string(http.basePath),
+          maxPayloadInBytes: http.maxPayload.getValueInBytes(),
+          rewriteBasePath: http.rewriteBasePath,
+          keepaliveTimeout: http.keepaliveTimeout,
+          socketTimeout: http.socketTimeout,
+          compression: {
+            enabled: http.compression.enabled,
+            referrerWhitelistConfigured: isConfigured.array(http.compression.referrerWhitelist),
+          },
+          xsrf: {
+            disableProtection: http.xsrf.disableProtection,
+            whitelistConfigured: isConfigured.array(http.xsrf.whitelist),
+          },
+          requestId: {
+            allowFromAnyIp: http.requestId.allowFromAnyIp,
+            ipAllowlistConfigured: isConfigured.array(http.requestId.ipAllowlist),
+          },
+          ssl: {
+            certificateAuthoritiesConfigured: isConfigured.stringOrArray(
+              http.ssl.certificateAuthorities
+            ),
+            certificateConfigured: isConfigured.string(http.ssl.certificate),
+            cipherSuites: http.ssl.cipherSuites,
+            keyConfigured: isConfigured.string(http.ssl.key),
+            redirectHttpFromPortConfigured: isConfigured.number(http.ssl.redirectHttpFromPort),
+            supportedProtocols: http.ssl.supportedProtocols,
+            clientAuthentication: http.ssl.clientAuthentication,
+            keystoreConfigured: isConfigured.record(http.ssl.keystore),
+            truststoreConfigured: isConfigured.record(http.ssl.truststore),
+          },
+        },
+
+        logging: {
+          appendersTypesUsed: Array.from(
+            Array.from(this.loggingConfig?.appenders.values() ?? [])
+              .reduce((acc, a) => acc.add(a.kind), new Set<string>())
+              .values()
+          ),
+          loggersConfiguredCount: this.loggingConfig?.loggers.length ?? 0,
+        },
+
+        savedObjects: {
+          maxImportPayloadBytes: this.soConfig.maxImportPayloadBytes.getValueInBytes(),
+          maxImportExportSizeBytes: this.soConfig.maxImportExportSize.getValueInBytes(),
+        },
+      },
+      environment: {
+        memory: {
+          heapSizeLimit: this.opsMetrics.process.memory.heap.size_limit,
+          heapTotalBytes: this.opsMetrics.process.memory.heap.total_in_bytes,
+          heapUsedBytes: this.opsMetrics.process.memory.heap.used_in_bytes,
+        },
+        os: {
+          platform: this.opsMetrics.os.platform,
+          platformRelease: this.opsMetrics.os.platformRelease,
+        },
```

Comment:
This is already part of the `kibana_stats` collector. But it makes more sense to me to be part of this collector. 

Anyway, you may want to consider adding the optional `distro` and `distroRelease` (provided on Linux environments).

Reply:
I've removed OS usage data. We want to use prettier names and it will be easier to add prettier names, move it over and remove the old usage data in a separate PR.